### PR TITLE
Circleci

### PR DESCRIPTION
**Pull request by Elexy**
add circleci config and root package.json

### CODE REVIEW

File: .circleci/config.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      #     name: code-coverage
+      #     command: './node_modules/.bin/nyc report --reporter=text-lcov'
+      # - store_artifacts:
+      #     path: test-results.xml
+      #     prefix: tests
+      # - store_artifacts:
+      #     path: coverage
+      #     prefix: coverage
+      # - store_test_results:
+      #     path: test-results.xml
```
Comment (mcollina) [inline .circleci/config.yaml]: if these steps are not needed, just remove them

File: package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "description": "Run benchmark tests on IPFS projects",
+  "main": "index.js",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "install": "cd tests && npm install && cd ../runner && npm install",
+    "test": "cd tests && npm run test && cd ../runner && npm run test",
+    "start": "LOG_PRETTY=true node runner/index.js",
+    "dev": "STAGE=local LOG_PRETTY=true node runner/index.js"
```
Comment (mcollina) [inline package.json]: I think we should not do this. instead add specific jobs for each part in circle config

File: .circleci/config.yaml (hunk 1 above)
Comment (Elexy) [reply .circleci/config.yaml]: yeah will remove them.

File: package.json (hunk 2 above)
Comment (Elexy) [reply package.json]: fixed

**Pull request merged**